### ISSUE TITLE
New Strapi 5 project with EE and mysql/mariadb fails to start

### ISSUE THREAD

**Issue by innerdvations** [strapi/strapi#21471]
### Required System information

Launched In: 666 ms
Environment: development
OS: darwin-arm64
Strapi Version: 5.0.0
Node/Yarn Version: yarn/4.5.0 npm/? node/v20.9.0 darwin arm64
Edition: Community
Database: mysql

### Describe the bug

Creating a new Strapi project with EE and mariadb or mysql fails to start the project with this error:

![image](https://github.com/user-attachments/assets/30940428-ab71-4d46-b272-4be65711f08b)

### Steps to reproduce the behavior

1. create a strapi 5 app
2. set an EE license key before starting it up the first time
3. start up strapi

### Expected behavior

Strapi should start up with no issues

### Workaround

- Disable EE
- start up strapi
- it creates all the initial tables it needs
- shut down strapi
- Enable EE

### Screenshots

![image](https://github.com/user-attachments/assets/30940428-ab71-4d46-b272-4be65711f08b)

### Code snippets

If applicable, add code samples to help explain your problem.

**Comment by derrickmehaffy** [comment]
Ben and I both confirmed this on MariaDB only when using an EE license

**Comment by yukimasaki** [comment]
I am using Strapi's Comunity Edition and I get the same message with MariaDB 10.6.

> Transaction was implicitly committed, do not mix transactions and DDL with MySQL (#805)

I can access the admin panel with `yarn develop`, but not with `yarn start`.
Chrome's developer tools shows the following error message.

![image](https://github.com/user-attachments/assets/d97cc2be-fb20-422c-8c88-52c8c672fa52)

Should I create a new Issue on this issue?

**Comment by hanpaine** [comment]
Hi @yukimasaki thank you for flagging - it would be great if you could raise another issue with as much further information as possible, yes please 🙏 cheers!
